content/_includes/page/list.njk

```diff
@@ -34,5 +34,62 @@
       'Events',
       collections.all | getCollectionItem(page) | pageEvents
     ) }}
+    {% if is_entry and ('Talk' in tags or 'Workshop' in tags) %}
+      <section data-list="tag" data-typeset>
```

Comment:
@sanajaved7 I don't know where to put this comment, but I noticed that even if nothing is shown on some pages like [work](https://deploy-preview-264--oddleventy.netlify.app/work/) and a [service page](https://deploy-preview-264--oddleventy.netlify.app/services/development/)  page, an empty 

```
<section data-list="tag" data-typeset></section>
```

is included in the markup

![Screen Shot 2022-11-08 at 2 06 31 PM](https://user-images.githubusercontent.com/1581694/200664763-0f309bbc-e9e1-422e-86ed-c35b18500ee8.jpg)